h/errors.py

```diff
@@ -0,0 +1,13 @@
+from pyramid.view import view_config
+
+
+@view_config(context=Exception, renderer='h:templates/5xx.html')
+def error(context, request):
+    """Display an error message and if necessary handle it otherwise."""
+    if hasattr(request.registry, 'handle_exception'):
```

Comment:
I don't like mingling the exception handling in here via the registry. There's a way that makes more sense using tweens, I think. Since I happen to be setting up some tweens for the authentication refactoring I'm doing, I'll push that here.

The idea is that pyramid has three named levels for tween ordering, INGRESS, EXCVIEW and MAIN.
So we just need a tween under the EXCVIEW that logs to sentry.